### PR TITLE
Add support for fitBounds on V2.0.X

### PR DESCRIPTION
**Pull request by echarlus**
After discussing my initial PR (#155) which was based on old code, I've re-implemented the changes on top of 2.0.3 version for Vaadin 23.3.
This change allows to automatically center & zoom the map based on a rectangular area which is useful if you want to optimize the view on a specific automatically

### CODE REVIEW

File: vaadin-maps-leaflet-flow-demo/pom.xml [hunk 1]
```diff
@@ -78,6 +78,14 @@
 					<scan>1</scan>
 				</configuration>
 			</plugin>
+			<plugin>
```
Comment (JohannesRabauer) [inline vaadin-maps-leaflet-flow-demo/pom.xml:81]: This change is uneccessary.

File: vaadin-maps-leaflet-flow/pom.xml [hunk 2]
```diff
@@ -231,6 +231,14 @@
 					</excludes>
 				</configuration>
 			</plugin>
+			<plugin>
```
Comment (JohannesRabauer) [inline vaadin-maps-leaflet-flow/pom.xml:234]: The component is packaged as a jar. There is no need to add the maven-war-plugin. Please delete this change.

File: vaadin-maps-leaflet-flow/src/main/java/software/xdev/vaadin/maps/leaflet/flow/LMap.java [hunk 3]
```diff
@@ -109,6 +109,14 @@ public void setViewPoint(final LCenter viewpoint)
 			+ viewpoint.getZoom()
 			+ ");");
 	}
+
+	public void centerAndZoom(final LPoint noPoint, final LPoint sePoint)
```
Comment (JohannesRabauer) [inline vaadin-maps-leaflet-flow/src/main/java/software/xdev/vaadin/maps/leaflet/flow/LMap.java:113]: Please add a minimal description. Maybe add the link to https://leafletjs.com/reference.html#map-fitbounds.
noPoint and sePoint seems not very descriptive.

File: vaadin-maps-leaflet-flow/src/main/java/software/xdev/vaadin/maps/leaflet/flow/data/LPoint.java [hunk 4]
```diff
@@ -26,18 +26,28 @@
 	
 	public LPoint(final double lat, final double lon)
 	{
-		this.coords.add(lat);
-		this.coords.add(lon);
+		coords.add(lat);
```
Comment (JohannesRabauer) [inline vaadin-maps-leaflet-flow/src/main/java/software/xdev/vaadin/maps/leaflet/flow/data/LPoint.java:29]: "this." should be added automatically through intellij. Please delete this change.

File: vaadin-maps-leaflet-flow/src/main/java/software/xdev/vaadin/maps/leaflet/flow/data/LPoint.java [hunk 5]
```diff
@@ -26,18 +26,28 @@
...
+		coords.add(lon);
```
Comment (JohannesRabauer) [inline vaadin-maps-leaflet-flow/src/main/java/software/xdev/vaadin/maps/leaflet/flow/data/LPoint.java:30]: "this." should be added automatically through intellij. Please delete this change.

File: vaadin-maps-leaflet-flow/src/main/java/software/xdev/vaadin/maps/leaflet/flow/data/LPoint.java [hunk 6]
```diff
@@ -26,18 +26,28 @@
...
 	
 	public List<Double> getCoords()
 	{
-		return this.coords;
+		return coords;
+	}
+	
+	public Double getLat()
+	{
+		return coords.get(0);
```
Comment (JohannesRabauer) [inline vaadin-maps-leaflet-flow/src/main/java/software/xdev/vaadin/maps/leaflet/flow/data/LPoint.java:40]: "this." should be added automatically through intellij. Please add "this." in front of coords.

File: vaadin-maps-leaflet-flow/src/main/java/software/xdev/vaadin/maps/leaflet/flow/data/LPoint.java [hunk 7]
```diff
@@ -26,18 +26,28 @@
...
+	}
+
+	public Double getLon()
+	{
+		return coords.get(1);
```
Comment (JohannesRabauer) [inline vaadin-maps-leaflet-flow/src/main/java/software/xdev/vaadin/maps/leaflet/flow/data/LPoint.java:45]: "this." should be added automatically through intellij. Please add "this." in front of coords.

**Review by JohannesRabauer — CHANGES_REQUESTED** [review]
Please fix these minor changes, then we can merge the branch. :)

**Comment by echarlus** [comment]
@JohannesRabauer I've pushed the requested changes to my develop branch. Let me know if it's ok.
Thanks

**Pull request merged**